picasso/src/main/java/com/squareup/picasso/PicassoDrawable.java

```diff
@@ -74,6 +80,8 @@ static void setPlaceholder(ImageView target, Drawable placeholderDrawable) {
       new BitmapDrawable(context.getResources(), bitmap)
     });
 
+    setCrossFadeEnabled(true);
```

Comment:
In the case of `TransitionDrawable` it always has to be set; otherwise it'll draw both layers when the "transition" is done.